contribs/gnogenesis/internal/txs/txs_add_packages.go

```diff
@@ -117,3 +131,55 @@ func execTxsAddPackages(
 
 	return nil
 }
+
+func signTxs(txs []gnoland.TxWithMetadata, signer *gnoclient.SignerFromKeybase) error {
+	for index, tx := range txs {
+		signBytes, err := tx.Tx.GetSignBytes(signer.ChainID, 0, 0)
+		if err != nil {
+			return fmt.Errorf("unable to load txs from directory, %w", err)
+		}
+		signature, publicKey, err := signer.Keybase.Sign(signer.Account, signer.Password, signBytes)
+		txs[index].Tx.Signatures = []std.Signature{
+			{
+				PubKey:    publicKey,
+				Signature: signature,
+			},
+		}
+		if err != nil {
+			return fmt.Errorf("unable sign tx %w", err)
+		}
+	}
+
+	return nil
+}
+
+func signerWithConfig(cfg *addPkgCfg, io commands.IO, chainID string) (*gnoclient.SignerFromKeybase, error) {
+	var (
+		keyname = integration.DefaultAccount_Name
+		pass    string
+		kb      keys.Keybase
+		err     error
+	)
+
+	if cfg.keyName != "" {
+		keyname = cfg.keyName
+		kb, err = keys.NewKeyBaseFromDir(cfg.gnoHome)
+		if err != nil {
+			return nil, fmt.Errorf("unable to load keybase: %w", err)
+		}
+		pass, err = io.GetPassword("Enter password.", cfg.insecurePasswordStdin)
+		if err != nil {
+			return nil, fmt.Errorf("cannot read password: %w", err)
+		}
+	} else {
+		kb = keys.NewInMemory()
+		kb.CreateAccount(integration.DefaultAccount_Name, integration.DefaultAccount_Seed, "", "", 0, 0)
```

Comment:
Why not handle this error?

Reply:
Because I forgot to do it :(
Thanks for the review c967c2f43104f565d44d7dcf789cfb7cdf1798e5